src/tests/token/test_erc20_votes.cairo

```diff
@@ -212,12 +213,13 @@ fn test_delegate_by_sig_hash_generation() {
     let nonce = 0;
     let expiry = 'ts2';
     let delegator = contract_address_const::<
-        0xe3338551aa36b2ed40c925c7c6b89ab27d21693829df63c3764cca3500448a
+        0x19dcd9e412145354a3328fb68b5975bded85972893eb42eed11355d4cfbb58a
```

Comment:
To use the same hash when computing the signature in the following test since that's the address of the account used.

Reply:
Note that these addresses change each time the code of the preset is updated, and this signatures/hashes should be manually updated, which is a pain. Foundry should address this by generating the signatures on the test itself instead of the hardcoding current requirement.

Reply:
Mmm I see. Yeah, sig gen on the test would be amazing